en/core-api/credit-card.md

```diff
@@ -86,32 +116,32 @@ var options = {
 MidtransNew3ds.getCardToken(cardData, options);
 ```
 
-You can use one of our test credentials for Card Payment:
+Use the following credentials to test the *Card* payment method.
 
 Name | Value
 --- | ---
-Card Number | `4811 1111 1111 1114`
-CVV | `123`
-Exp Month | Any month (e.g: `02`)
-Exp Year | Any future year (e.g: `2025`)
-OTP/3DS | `112233`
```

Comment:
Do not remove the formatting (the backtick) please, it helps to shows which is the `value`, and which is not the literal value. Also do not remove the example of
```
(e.g: `02`)
```
it gives exact sample of the value.

Reply:
This is not yet resolved too